spec/requests/users_spec.rb

```diff
@@ -1,7 +1,25 @@
 require 'rails_helper'
 
-RSpec.describe 'Users', type: :request do
-  describe 'GET /index' do
-    pending "add some examples (or delete) #{__FILE__}"
+RSpec.describe UsersController, type: :request do
+  describe 'Request spec for users index' do
+    it 'returns a successful response' do
+      puts 'Checking HTTP status for users index...'
+      get '/users'
+      expect(response).to have_http_status(200)
+    end
+  end
```

Comment:
- [ ] Kindly add a test that ensures the response body includes the correct placeholder text for the index page of the User. Example : `expect(response.body).to include("Welcome to user index page")`